### ISSUE TITLE
Make table caption a sibling of the table tag

### ISSUE THREAD

**Issue by pcraig3** [HHS/simpler-grants-pdf-builder#107]
> The table caption tag needs to be moved up to a sibling of the Table tag

| before | after  |
|--------|--------|
|  Caption element inside table      |  No caption element, just a paragraph before the table       |
| <img width="506" alt="Image" src="https://github.com/user-attachments/assets/ac940299-61d4-4cfb-961e-433574a72c63" /> | <img width="514" alt="Image" src="https://github.com/user-attachments/assets/5fcd8770-ff21-475a-8ee0-219df06b499c" /> |

Note: not clear that this is 100% what we should do.

The HTML `<caption>` should always be the first child of a table: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption

Similarly, it seems like there is guidance out there that `<Caption>` should be the first child of a `<Table>` in a PDF: https://community.adobe.com/t5/acrobat-discussions/lt-caption-gt-and-lt-table-gt-tag/m-p/9907424

**Comment by pcraig3** [comment]
I am going to make an executive decision to close this until we hear otherwise.
